tests/integ/modin/frame/test_apply.py

```diff
@@ -60,45 +60,10 @@
     ),
 ]
 
-# test data where return type is a pandas Series
-BASIC_DATA_FUNC_SERIES_RETURN_TYPE_MAP = [
-    [[[1.1, 2.2], [3, np.nan]], lambda x: x + 1, "native_pd.Series"],
-    [[[1.1, 2.2], [3, np.nan]], np.sqrt, "native_pd.Series"],
-    [[[1.1, 2.2], [3, np.nan]], lambda x: x > 2, "native_pd.Series"],
-    [
-        [["snow", "flake"], ["data", "cloud"]],
-        lambda x: x.str.replace("a", "b"),
-        "native_pd.Series",
-    ],
-    [[[True, False], [False, False]], lambda x: x.astype(np.int64), "native_pd.Series"],
-    [[[True, False], [False, False]], lambda x: x[0] ^ x[1], "bool"],
-    (
-        [
-            [bytes("snow", "utf-8"), bytes("flake", "utf-8")],
-            [bytes("data", "utf-8"), bytes("cloud", "utf-8")],
-        ],
-        lambda x: x.astype(str),
-        "native_pd.Series",
-    ),
-    (
-        [[["a", "b"], ["c", "d"]], [["a", "b"], ["c", "d"]]],
-        lambda x: x,
-        "native_pd.Series",
-    ),
-    (
-        [[{"a": "b"}, {"c": "d"}], [{"c": "b"}, {"a": "d"}]],
-        lambda x: x,
-        "native_pd.Series",
-    ),
-]
 
-BASIC_DATA_FUNC_RETURN_TYPE_MAP = (
-    BASIC_DATA_FUNC_PYTHON_RETURN_TYPE_MAP + BASIC_DATA_FUNC_PYTHON_RETURN_TYPE_MAP
```

Comment:
Instead of removing, why not do 
`BASIC_DATA_FUNC_RETURN_TYPE_MAP = ( BASIC_DATA_FUNC_SERIES_RETURN_TYPE_MAP + BASIC_DATA_FUNC_PYTHON_RETURN_TYPE_MAP)`

Reply:
It does not improve our test though. E.g., there is a field called `return type` in the list but it is never used in the tests.